src/main/java/org/reaktivity/nukleus/kafka/internal/stream/KafkaCacheServerProduceFactory.java

```diff
@@ -474,8 +531,8 @@ private void doServerFanInitialBegin(
                 ex -> ex.set((b, o, l) -> kafkaBeginExRW.wrap(b, o, l)
                         .typeId(kafkaTypeId)
                         .produce(p -> p.transaction(TRANSACTION_NONE)
-                                       .topic(partition.topic())
-                                       .partitionId(partition.id()))
+                                       .topic(partionTopic)
+                                       .partition(par -> par.partitionId(partitionId).partitionOffset(DEFAULT_LATEST_OFFSET)))
```

Comment:
Suggest `part` instead of `par` here, or else `pr` instead of `p` for `produce(...)` lambda, leaving `p` for partition lambda.